saleor/checkout/models.py

```diff
@@ -46,7 +46,7 @@ class Checkout(ModelWithMetadata):
         related_name="checkouts",
         on_delete=models.CASCADE,
     )
-    email = models.EmailField()
+    email = models.EmailField(blank=True, default="")
```

Comment:
Maybe instead of `default=""` we should use `null=True` so that we have explicit null in the database when email is not set, and we don't return an empty string in API, but null.